src/rpp/timer.h

```diff
@@ -4,7 +4,10 @@
  * Distributed under MIT Software License
  */
 #include "config.h"
-#include <string>
+
+#ifndef BARE_METAL
```

Comment:
The macro should be defined in "config.h" if possible, and should be always set to either 1 or 0. Prefix `RPP_` is used.
Then usage is typically:
```cpp
#if !RPP_BARE_METAL
```